Microsoft.Xrm.Data.PowerShell/Microsoft.Xrm.Data.PowerShell.psm1

```diff
@@ -3943,6 +3943,74 @@ function Invoke-CrmWhoAmI{
     return $result
 }
 
+function Invoke-CrmAction {
+# .ExternalHelp Microsoft.Xrm.Data.PowerShell.Help.xml
+    [OutputType([hashtable])]
+    [OutputType([Microsoft.Xrm.Sdk.OrganizationResponse], ParameterSetName="Raw")]
+    param (
+        [Microsoft.Xrm.Tooling.Connector.CrmServiceClient]
+        $conn,
+
+        [Parameter(
+            Position=1,
+            Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $Name,
+
+        [Parameter(ValueFromPipeline)]
+        [ValidateNotNullOrEmpty()]
+        [Microsoft.Xrm.Sdk.EntityReference]
+        $Target,
+
+        [Parameter(Position=2)]
+        [ValidateNotNullOrEmpty()]
+        [hashtable]
+        $Parameters,
+
+        [Parameter(ParameterSetName="Raw")]
+        [switch]
+        $Raw
```

Comment:
I used Raw to mimic what [Get-Content -Raw](https://msdn.microsoft.com/en-us/powershell/reference/5.1/microsoft.powershell.management/get-content?f=255&MSPPError=-2147217396#-raw) does to get the full contents of a file in a single object instead of the default array, and figured it might be consistent.